tests/src/liberty/tools/test/it/LibertyPluginSWTBotMavenTest.java

```diff
@@ -448,6 +454,41 @@ public void validateTestReportExists(Path pathToTestReport) {
         Assertions.fail("Timed out while waiting for test report: " + pathToTestReport + " to become available.");
     }
 
+    /**
+     * Validates the content of an editor based on the input validation text.
+     *
+     * @param editorTitle The editor title of the editor to check.
+     * @param validationText The text to use for validation.
+     * @param contains The validation type. If true, the code checks that the editor contains the validation text. If false,
+     *        the code checks that the editor does not contain the validation text.
+     */
+    public void validateEditorContent(String editorTitle, String validationText, boolean contains) {
+        int retryCountLimit = 20;
+        int reryIntervalSecs = 1;
+        int retryCount = 0;
+        String editorContent = "";
+        while (retryCount < retryCountLimit) {
+            retryCount++;
+            editorContent = SWTPluginOperations.getEditorText(bot, editorTitle);
+
+            if ((!contains && editorContent.contains(validationText)) || (contains && !editorContent.contains(validationText))) {
```

Comment:
I get it for contains=true.

But  contains=false it seems less clear.  I guess it depends on what the condition is that we iterate through the while loop. 

Like if one pass through only got me half the input I should be checking, I might say "OK, good, it doesn't contain the bad text" and pass the check, but maybe I should be reading the next iteration's worth of input.    I.e. do I need to check that I've gotten the "full" editorContent input before doing a !contains check?

Reply:
If the validation type `contains` equals false, the code will make sure that the validation text is gone from the file; otherwise, it re-tries the check again. In the context of the test, we overwrite the contents of the original file (the overwrite has less entries than the original) and we hit save. Since all of the processing is async, there is some chance that by the time we execute the next test command (refresh), the file is not really saved yet. It reads the full content of the file before it checks. 
Let me know, if you still have an issue with this.